lib/phlex/html.rb

```diff
@@ -131,6 +131,10 @@ class HTML
 
 		class << self
 			attr_accessor :rendered_at_least_once
+
+			def render(...)
+				new(...).call
```

Comment:
The `...` here won't work in `2.7`.

Reply:
Wasn't Argument Forwarding introduced in 2.7?

Reply:
Oh, you're right.